net/netcheck/netcheck.go

```diff
@@ -861,6 +972,11 @@ func (c *Client) logConciseReport(r *Report, dm *tailcfg.DERPMap) {
 		fmt.Fprintf(w, " v6=%v", r.IPv6)
 		fmt.Fprintf(w, " mapvarydest=%v", r.MappingVariesByDestIP)
 		fmt.Fprintf(w, " hair=%v", r.HairPinning)
+		if r.UPnP != "" || r.PMP != "" || r.PCP != "" {
```

Comment:
Did this in a couple places. `hasPortmapServices()` helper on the report obj maybe?

Reply:
Done.